src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,71 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Calculator {
+
+    private final String DEFAULT_DELIMITER = "[,:]";
+    private final int LAST_PREFIX_INDEX = 4;
+    private final int CUSTOM_DELIMITER_INDEX = 2;
+
+    public int sumAll(String input) {
```

Comment:
```suggestion
    public int sum(String input) {
```

중요하지 않은 의견이지만 sum이 가진 의미가 "합산"이라 All까지는 안붙여도 될 것 같습니다.
굳이 일상생활과 비유를 해보면 역전앞 같은 느낌이랄까요.

Reply:
getSum 메서드에서 sum 이라는 지역 변수를 사용했어서 같은 이름의 메서드를 안쓰려다 보니 sumAll 이라는 이름을 사용했는데, 말씀하신것처럼 중복된 의미가 맞다고 생각해서 sum 으로 수정했습니다!!